### PR TITLE
Fix invalid resolution in lodash.merge

### PR DESCRIPTION
**Pull request by petereast**
#### Background

What does this PR do?
It fixes a version resolution error in the api-tests package.json

#### Any relevant tickets

Please use the phrase 'Closes' if it will fully complete the ticket.

#### How has this been tested?
**Reviewers** ensure that these test requirements are also reviewed.

Please indicate the need for this change to be tested. If not please justify, if so then at what levels. 

- [ ] Unit / Integration tests
- [ ] Browser tests
- [ ] End2end tests

#### UI Changes

- [ ] Has been reviewed against Designs
- [ ] Necessary updates to styleguide

### CODE REVIEW

**Review by diversemix — APPROVED** [review]

File: tools/api-tests/yarn.lock [hunk 1]
```diff
@@ -65,41 +65,41 @@
     imurmurhash "^0.1.4"
```
Comment (giorgiosironi) [inline tools/api-tests/yarn.lock:65]: this file appears not to be in use? https://github.com/elifesciences/elife-xpub/pull/2350

**Review by giorgiosironi — APPROVED** [review]
Doesn't hurt

**Review by giorgiosironi — COMMENTED** [review]
*But it upgrades lots of (unrelated?) packages

File: tools/api-tests/yarn.lock [hunk 2]
```diff
@@ -320,33 +321,32 @@
     arrify "^1.0.1"
 
 "@graphql-codegen/cli@^1.2.0":
-  version "1.2.0"
-  resolved "https://registry.yarnpkg.com/@graphql-codegen/cli/-/cli-1.2.0.tgz#5a67c817559783c0d2e68199ab98f9003188f911"
-  integrity sha512-X/CkvcEDBSmwflFg7ctFJYu/GLw6Q4XfMj3Xgqlj+tT0OV9eLZM+jW//e9T0Kan3JxcZDFBQpA0gULyIcLlWdg==
+  version "1.5.0"
```
Comment (giorgiosironi) [inline tools/api-tests/yarn.lock:324]: do we want this update? it's the suspect for the build problems

**Review by giorgiosironi — CHANGES_REQUESTED** [review]
Tests pass locally with `@graphql-codegen/cli` `1.2.0` so probably should not be upgraded

**Comment by giorgiosironi** [comment]
Closing in favor of https://github.com/elifesciences/elife-xpub/pull/2350

**Pull request closed without merging**